src/rules/commentFormatRule.ts

```diff
@@ -39,20 +39,25 @@ export class Rule extends Lint.Rules.AbstractRule {
         options: {
             type: "array",
             items: {
-                type: "string",
-                enum: ["check-space", "check-lowercase", "check-uppercase"],
+                oneOf: [{
+                    type: "string",
+                    enum: ["check-space", "check-lowercase", "check-uppercase"],
+                }, {
+                    type: "array",
```

Comment:
would it be better to just make this a single regex string since the code makes it into a reg expression anyways? Also, people might see weird behavior if they don't know it's a regex.